addons/advanced_fatigue/CfgEden.hpp

```diff
@@ -3,7 +3,7 @@ class Cfg3DEN {
         class Slider;
         class GVAR(slider): Slider {
             #pragma hemtt suppress pw3_padded_arg
-            attributeLoad = QUOTE(
+            attributeLoad = QUOTE( \
```

Comment:
Everywhere else we have this structure we don't have a space
```suggestion
            attributeLoad = QUOTE(\
```